db/schema.rb

```diff
@@ -432,7 +467,7 @@
   add_index "scores", ["response_id", "question_id"], name: "index_scores_on_response_id_and_question_id", unique: true, using: :btree
 
   create_table "sessions", force: :cascade do |t|
-    t.string   "session_id", null: false
+    t.string   "session_id", limit: 255, null: false
```

Comment:
I have no idea what happened here, I'd try to restore the schema from staging and reapply migrations.